frontend/portals/GlobalCssRules/index.jsx

```diff
@@ -0,0 +1,7 @@
+import { css } from 'glamor';
+
+css.global('.avg-score', {
```

Comment:
instead of a global styling it should be applied here https://github.com/shopgate-professional-services/ext-yotpo-reviews/blob/master/frontend/components/YotpoMainHtmlReferenceContainer/components/YotpoMainWidget/index.jsx#L64 
With this is automatic scoped for this container only